libs/agno/agno/os/app.py

```diff
@@ -369,6 +380,37 @@ def _initialize_workflows(self) -> None:
                 # Required for the built-in routes to work
                 workflow.store_events = True
 
+    def _setup_tracing(self) -> None:
+        """Set up OpenTelemetry tracing for this AgentOS."""
+        # Find the first available database
```

Comment:
Don't we want to call setup_tracing for all available dbs?

Also if we call _setup_tracing after _auto_discover_databases we can avoid iterating over agents etc, and just read self.dbs -> that means calling this in the get_app logic instead of the init, but that's okay